ports/espressif/boards/m5stack_stamp_c3/mpconfigboard.mk

```
@@ -0,0 +1,10 @@
+CIRCUITPY_CREATOR_ID =  0x10151015
+CIRCUITPY_CREATION_ID = 0x00C30001
+
+IDF_TARGET = esp32c3
+
+INTERNAL_FLASH_FILESYSTEM = 1
+
+CIRCUITPY_ESP_FLASH_MODE=qio
```

```suggestion
CIRCUITPY_ESP_FLASH_MODE=qio
```
Minor change request, `INTERNAL_FLASH_FILESYSTEM = 1` is already set in `mpconfigport.mk`.